core/src/main/java/jenkins/security/csrf/CSRFAdministrativeMonitor.java

```diff
@@ -0,0 +1,37 @@
+package jenkins.security.csrf;
```

Comment:
Seems a bit much to create a separate package for this, no?

Notably, crumb issuers and related existing code cannot be moved anyway.

Reply:
Usually we create mirror packages in Jenkins for the new things. I can imagine new classes around csrf things, so having a new package LGTM.

Reply:
Ah, I missed that `hudson.security.csrf` exists. 👍 then